tests/regression/lp-1819728/task.yaml

```
@@ -0,0 +1,28 @@
+summary: Ensure that the systemctl from PR#8803 is fixed
+
+description: |
+    See https://bugs.launchpad.net/ubuntu/+source/systemd/+bug/1819728
+    for details about this issue.
+
+# TODO: enable on ubuntu-16.04-64, ubuntu-18.04-64 once the SRU
+#       for systemd is in the "-updates" pocket.
+systems: [ubuntu-core-16-64, ubuntu-core-18-64]
+
+kill-timeout: 2m
+
+execute: |
+    echo "When running a bunch of systemctl commands and daemon-reloads"
+    for _ in $(seq 50); do
+        systemctl daemon-reload &
+        systemctl start ssh &
+    done
+
+    echo "Then the system will eventually recover and no systemctl are left"
+    for _ in $(seq 10); do
+        if ! pgrep systemctl; then
+            break
+        fi
+        sleep 6
+    done
+    echo "No systemctl left"
+    ! pgrep systemctl
```

Perhaps add something like:
```suggestion
    ! pgrep systemctl
debug: |
    ps aux | grep systemctl
```